src/components/mainPage/mainPage.tsx

```diff
@@ -4,7 +4,17 @@ class ProofEditor extends React.Component {
 
 	public render() {
 		return <div className={"addStyleHere"}>
```

Comment:
className can be removed. I think this was inserted to just mark a possible stylesheet location, but it doesn't do anything as of now.
I might have even inserted it back then, but I am not sure anymore.